Logging/composer.json

```
@@ -12,23 +12,21 @@
         "yoast/phpunit-polyfills": "^1.0",
         "phpspec/prophecy": "^1.10.3",
         "squizlabs/php_codesniffer": "2.*",
-        "phpdocumentor/reflection": "^3.0||^4.0",
+        "phpdocumentor/reflection": "^3.0||^4.0||^5.3",
         "erusev/parsedown": "^1.6",
+        "fig/log-test": "^1.0",
         "google/cloud-storage": "^1.3",
         "google/cloud-bigquery": "^1.0",
         "google/cloud-pubsub": "^1.0",
         "opis/closure": "^3"
     },
     "provide": {
-        "psr/log-implementation": "1.0|2.0"
+        "psr/log-implementation": "^1.0|^2.0|^3.0"
```

This is not accurate because `PsrLogger` is not compatible with `psr/log: 3.0` (yet)
```suggestion
        "psr/log-implementation": "^1.0|^2.0"
```

---

I just reverted and pushed the changes